File: 2313/2313.cs
public class Solution {
    public int MinimumFlipsInBinaryTreeToGetR(TreeNode root) {
        if (root == null) return 0;
        return 1 + Math.Max(MinimumFlipsInBinaryTreeToGetR(root.left), MinimumFlipsInBinaryTreeToGetR(root.right));
    }
}

File: 2313/2313.go
func minimumFlipsInBinaryTreeToGetR(root *TreeNode) int {
	if root == nil {
		return 0
	}
	l := minimumFlipsInBinaryTreeToGetR(root.Left)
	r := minimumFlipsInBinaryTreeToGetR(root.Right)
	if l > r {
		return 1 + l
	}
	return 1 + r
}